compiler/noirc_evaluator/src/ssa.rs

```diff
@@ -70,6 +70,9 @@ pub struct SsaEvaluatorOptions {
 
     /// The higher the value, the more inlined brillig functions will be.
     pub inliner_aggressiveness: i64,
+
+    /// Collect extra debug information for more informative profiling.
+    pub profiling_active: bool,
```

Comment:
Is this necessary? It doesn't seem like it's a ton of extra work to add this debug info (plus it's at compile time rather than at execution).

Unless the debug info generated is truly huge I think that we can just always generate debug data for procedures.

Reply:
It isn't a ton of extra work, it just may get large if we have many Brillig functions. But either way it wouldn't be _that_ large. A maximum of two usizes to represent the procedure range for each procedure for each function.

Reply:
I'll remove the flag

Reply:
Removed